code/go/0chain.net/chaincore/transaction/worker.go

```diff
@@ -62,7 +62,16 @@ func CleanupWorker(ctx context.Context) {
 				logging.Logger.Error("Error in IterateCollectionAsc", zap.Error(err))
 			}
 			if len(invalidTxns) > 0 {
-				logging.Logger.Info("transactions cleanup", zap.String("collection", collectionName), zap.Int("invalid_count", len(invalidTxns)), zap.Int64("collection_size", mstore.GetCollectionSize(cctx, transactionEntityMetadata, collectionName)))
+				invalidTxnHashes := make([]string, len(invalidTxns))
+				for i, t := range invalidTxns {
+					invalidTxnHashes[i] = t.(*Transaction).Hash
+				}
+
+				logging.Logger.Info("transactions cleanup",
+					zap.String("collection", collectionName),
+					zap.Int("invalid_count", len(invalidTxns)),
+					zap.Any("txns", invalidTxnHashes),
```

Comment:
we can avoid logging any

Reply:
Sure, I'll change to strings or remove it perhaps